pkg/webhook/webhook_test.go

```diff
@@ -356,83 +373,308 @@ func TestGitHubWrongSecret(t *testing.T) {
 	}
 }
 
-func TestGitHubRightSecretAndCommitUpdated(t *testing.T) {
+func TestGitHubSecretAndCommitUpdated(t *testing.T) {
 	expectedCommit := "af69d162de5a276abc86e0686b2b44033cd3f442"
-
-	ctlr := gomock.NewController(t)
-	mockClient := mocks.NewMockClient(ctlr)
-
-	gitRepo := &v1alpha1.GitRepo{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "test",
+	gitrepoSecretName := "gitrepoSecret"
+	tests := map[string]struct {
+		secretValueInWebhook string
```

Comment:
This field doesn't seem to be used :thinking: 
Should it be used instead of a hard-coded string on L641 maybe? (and ideally renamed to `secretValueInRequest`, or something similar to clarify that the secret is to be found in the request received from the webhook provider)

Reply:
changed as suggested and using it instead of the hardcoded one